var/spack/repos/builtin/packages/git/package.py

```diff
@@ -167,6 +167,15 @@ def filter_rt(self):
             filter_file(r' -lrt$', '', 'Makefile')
 
     @run_after('install')
+    def install_completions(self):
+        prefix = self.prefix
+        mkdirp(prefix.share)
+        with working_dir('contrib/completion'):
+            for f in ['git-completion.bash', 'git-completion.tcsh',
+                      'git-completion.zsh', 'git-prompt.sh']:
+                install(f, join_path(prefix.share, f))
```

Comment:
`install_tree` currently requires that the installation directory doesn't exist. I would like to fix that in #3701. In the meantime, remove the `mkdirp` and see if that works. If `prefix.share` is already populated by `make install`, you'll have to use `copy_tree` instead:
```python
from distutils.dir_util import copy_tree

copy_tree('contrib/completion', self.prefix.share)
```